src/components/ConnectToXeroButton/index.native.tsx

```diff
@@ -0,0 +1,101 @@
+import React, {useRef, useState} from 'react';
+import type {OnyxEntry} from 'react-native-onyx';
+import {withOnyx} from 'react-native-onyx';
+import WebView from 'react-native-webview';
```

Comment:
I haven't tested this yet, but the QBO button implementation and the `react-native-webview` [docs usage](https://github.com/react-native-webview/react-native-webview?tab=readme-ov-file#usage) shows the import as:

`import {WebView} from 'react-native-webview';`